### PR TITLE
WIP: completing draft of first blog as official documentation

### PR DESCRIPTION
**Pull request by MBishop17**
I finished a draft of official docs for Prachi's first blog. Please take a look and offer up some feedback before I continue on additional blogs.

### CODE REVIEW

File: content/rancher/v2.x/en/v1.6-migration/get-started/_index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+- [C—Provision a Cluster and Project](#cprovision-a-cluster-and-project)
+- [D—Create Namespaces](#dcreate-namespaces)
+
+<!-- /TOC -->
+
+## A—Install Rancher 2.0
+
+The first step in migrating from 1.6 to 2.0 is to install the Rancher 2.0 Server. Due the architecture changes between 1.6 and 2.0, there is no direct path for upgrade. You'll have to install Rancher 2.0 independently and then migrate your 1.6 content to 2.0.
+
+New for Rancher 2.0, all communication with Rancher Server is encrypted. This encryption requires you to install an SSL certificate. The procedures below will instruct you on how to create and install these certificates.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/v1.6-migration/get-started/_index.md]: I'd rephrase to `all communication to Rancher server` as the traffic between nodes in a user cluster is not unless you are choosing a networking option that has it encrypted.

File: content/rancher/v2.x/en/v1.6-migration/get-started/_index.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    For development environments, we recommend a single node install. This installation procedure deploys a single Rancher container to your host.
+    
+- [High Availability Configuration]({{< baseurl >}}/rancher/v2.x/en/installation/ha/)
+
+    For production environments where your user base requires consistent access to your cluster, we recommend installing Rancher in a high availability configuration. This installation procedure provisions a three-node cluster using a Helm chart. Rancher is installed on each node.
+
+## B—Configure Authentication
+
+After Rancher Server is installed, we recommend configuring external authentication so that users can log in using their single sign-on.
+
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/v1.6-migration/get-started/_index.md]: We should make a note about some small differences in 1.6 versus 2.0 for auth providers.

1. In 1.6, for the SAML based auth providers, we directed people to use the Shibboleth auth provider, but in 2.0 we have revised it so that each SAML auth provider is tested and a new icon is made due to small nuances between the auth providers. 

2. Also, in 1.6, you can configure and have local authenticated users in addition to an external authentication provider.

File: content/rancher/v2.x/en/v1.6-migration/get-started/_index.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+After provisioning your node(s), install Rancher:
+
+- [Single Node Install]({{< baseurl >}}/rancher/v2.x/en/installation/single-node)
+    
+    For development environments, we recommend a single node install. This installation procedure deploys a single Rancher container to your host.
+    
+- [High Availability Configuration]({{< baseurl >}}/rancher/v2.x/en/installation/ha/)
+
+    For production environments where your user base requires consistent access to your cluster, we recommend installing Rancher in a high availability configuration. This installation procedure provisions a three-node cluster using a Helm chart. Rancher is installed on each node.
+
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/v1.6-migration/get-started/_index.md]: I'd make a note about how the HA install requires having Rancher 2.0 installed on a Kubernetes cluster unlike in 1.6, you just needed to do a `docker run` with additional requirements.

**Comment by deniseschannon** [comment]
@MBishop17 overall looks good. just some small additional notes that i think we should add to give users understanding about the diff between 1.6 and 2.0.

File: content/rancher/v2.x/en/v1.6-migration/get-started/_index.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+[3]:{{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/github
+[4]:{{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/openldap
+[5]:{{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/ping-federate
+[6]:{{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/microsoft-adfs
+[7]:{{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/freeipa
+
+### Authentication Note: Local Users
+
+Although we recommend using an external authentication provider, Rancher 1.6 and 2.0 both offer support for users local to Rancher. However, these users cannot be migrated from Rancher 1.6 to 2.0. If you used local users for Rancher 1.6 and want to continue that practice for 2.0, you'll need to [manually recreate these user accounts]({{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/) and assign them access rights.
+
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/v1.6-migration/get-started/_index.md]: We should also add a note about how you can have local users *and* an external auth provider as login options.

**Comment by MBishop17** [comment]
closing for #1012
